dev-packages/application-manager/src/application-package-manager.ts

```diff
@@ -182,7 +182,15 @@ export class ApplicationPackageManager {
             throw new AbortError('Dependencies are out of sync, please run "install" again');
         }
         await ffmpeg.replaceFfmpeg();
-        await ffmpeg.checkFfmpeg();
+        try {
+            await ffmpeg.checkFfmpeg();
+        } catch (error) {
+            if (error.message.includes('(have \'arm64\', need \'x86_64\')') && process.env.ARCH === 'arm64') {
+                console.warn('WARN: ffmpeg checking skipped because mac-o file incompatible in this step');
```

Comment:
I believe it should be `mach-o`?

```suggestion
                console.warn('WARN: ffmpeg checking skipped because mach-o file incompatible in this step');
```

The error is:


```
dyld[54558]: Library not loaded: @rpath/libffmpeg.dylib
  Referenced from: /Users/alipinar/Desktop/workspace/theia-blueprint/node_modules/electron/dist/Electron.app/Contents/Frameworks/Electron Framework.framework/Versions/A/Electron Framework
  Reason: tried: '/Users/alipinar/Desktop/workspace/theia-blueprint/node_modules/electron/dist/Electron.app/Contents/Frameworks/Electron Framework.framework/Versions/A/Libraries/libffmpeg.dylib' (mach-o file, but is an incompatible architecture (have 'x86_64', need 'arm64e')), '/Users/alipinar/Desktop/workspace/theia-blueprint/node_modules/electron/dist/Electron.app/Contents/MacOS/../Frameworks/libffmpeg.dylib' (no such file), '/Users/alipinar/Desktop/workspace/theia-blueprint/node_modules/electron/dist/Electron.app/Contents/MacOS/../Frameworks/libffmpeg.dylib' (no such file), '/usr/local/lib/libffmpeg.dylib' (mach-o file, but is an incompatible architecture (have 'x86_64', need 'arm64e')), '/usr/lib/libffmpeg.dylib' (no such file)
/Users/alipinar/Desktop/workspace/theia-blueprint/node_modules/electron/dist/Electron.app/Contents/MacOS/Electron exited with signal SIGABRT
error Command failed with exit code 1.
```